auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.kt

```diff
@@ -95,9 +95,12 @@ public class CredentialsManager @VisibleForTesting(otherwise = VisibleForTesting
         if (!hasEitherExpired && !willAccessTokenExpire && !scopeChanged) {
             callback.onSuccess(
                 recreateCredentials(
-                    idToken, accessToken, tokenType, refreshToken, Date(
-                        expiresAt
-                    ), storedScope
+                    idToken.orEmpty(),
```

Comment:
handles migration from "legacy credentials" passing empty strings when values were not previously saved